pythonforandroid/bootstraps/pygame/__init__.py

```diff
@@ -40,21 +43,14 @@ def run_distribute(self):
             info('Copying python distribution')
             hostpython = sh.Command(self.ctx.hostpython)
             # AND: This *doesn't* need to be in arm env?
-            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir())
+            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir(),
+                    _tail=10, _filterout="^Listing", _critical=True)
```

Comment:
1 its ok to drop patchall
2 presplash it was just experiment, actually no presplash code for sdl2 so
it is my fault to commit unused image
3. some commands produce really long outputs > 1000 lines. it was a big
problem for me to find error reason.
actually, in current state of p4a there no stdout listing at all (even id
debug) - just exception traceback and shortened last line. so, 10 lines is
much better then zero ?
 anyway feel free to increase it to any reasonable number of lines you want
:)

2015-11-27 0:56 GMT+02:00 Alexander Taylor <notifications@github.com>:

> In pythonforandroid/bootstraps/pygame/__init__.py
> <https://github.com/kivy/python-for-android/pull/480#discussion_r46009546>
> :
>
> > @@ -40,21 +43,14 @@ def run_distribute(self):
> >              info('Copying python distribution')
> >              hostpython = sh.Command(self.ctx.hostpython)
> >              # AND: This *doesn't* need to be in arm env?
> > -            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir())
> > +            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir(),
> > +                    _tail=10, _filterout="^Listing", _critical=True)
>
> Do you have a reason for limiting to the last 10 lines? If we're passing
> the output through to the user, don't we want to pass all of it (or, if
> debug is not enabled, none of it)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kivy/python-for-android/pull/480/files#r46009546>.
>